opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -208,4 +208,11 @@ message LogRecord {
   //   - the field is not present,
   //   - the field contains an invalid value.
   bytes span_id = 10;
+
+  // A globally unique identifier of event category/type among all events. All
+  // events with the same event_name MUST conform to the same schema for both their
```

Comment:
I'm not sure how to think about "MUST be globally unique" on an attribute value that we don't control

```suggestion
  // events with the same event_name are expected to conform to the same schema for both their
```